### PR TITLE
Update Postman collection for v0.30.0

### PR DESCRIPTION
**Pull request by curquiza**
Added:
- pagination
- task cancel
- task deletion
- index swap

updated
- some filter naming

### CODE REVIEW

File: .vuepress/public/postman/meilisearch-collection.json [hunk 1]
```diff
@@ -374,20 +401,144 @@
...
+									"value": null,
+									"disabled": true
+								},
+								{
+									"key": "afterFinishedAt",
+									"value": null,
+									"disabled": true
+								},
+								{
+									"key": "canceledBy",
```
Comment (maryamsulemani97) [inline .vuepress/public/postman/meilisearch-collection.json:538]: Can we use `canceledBy` with `/tasks/cancel?`?

Comment (curquiza) [reply .vuepress/public/postman/meilisearch-collection.json:538]: Yes, see [the spec](https://github.com/meilisearch/specifications/pull/195/files) and the [internal question](https://meilisearch.slack.com/archives/C044GEXNEUC/p1669112647316479) I asked following your question 😇

Comment (maryamsulemani97) [reply .vuepress/public/postman/meilisearch-collection.json:538]: Thanks!

**Review by maryamsulemani97 — APPROVED** [review]
Thanks @curquiza 🍪

**Comment by maryamsulemani97** [comment]
bors merge

**Pull request merged**